packages/playwright-core/src/utils/registry.ts

```diff
@@ -709,16 +709,17 @@ function markerFilePath(browserDirectory: string): string {
   return path.join(browserDirectory, 'INSTALLATION_COMPLETE');
 }
 
-export function buildPlaywrightCLICommand(sdkLanguage: string, parameters: string): string {
+export function buildPlaywrightCLICommand(sdkLanguage?: string, parameters?: string): string {
+  const parametersWithSpacePrefix = parameters ? (' ' + parameters) : '';
   switch (sdkLanguage) {
     case 'python':
-      return `playwright ${parameters}`;
+      return `playwright${parametersWithSpacePrefix}`;
     case 'java':
-      return `mvn exec:java -e -Dexec.mainClass=com.microsoft.playwright.CLI -Dexec.args="${parameters}"`;
+      return `mvn exec:java -e -Dexec.mainClass=com.microsoft.playwright.CLI -Dexec.args="${parameters ? parameters : '...options..'}"`;
```

Comment:
What is this '...options..'? Why don't we omit "exec.args" entirely?

Reply:
I thought about it. Then Maven users don't know where to put the args, since in Java its non-trivial in my mind.

Do you have a better proposal instead of `...options...`?

Reply:
Oh, I see. I'd prefer this logic to be in the CLI instead of the generic utility then.